### ISSUE TITLE
TV Search By Network

### ISSUE THREAD

**Issue by bbakermmc** [pkkid/python-plexapi#402]
The search network = network seems to return everything.  Not sure if Im using it right?
```python
NETWORKS = ["Disney+", "Apple TV+", "Netflix"]

for networkName in NETWORKS:
    print(networkName)
    for playlist in plex.playlists():
        if playlist.title == networkName:
            r = requests.delete('{}/playlists/{}?X-Plex-Token={}'.format(baseurl, networkName, token))
            print('{} already exists. Deleting it and will rebuild.'.format(networkName))

tv_shows_section: Union[ShowSection, PhotoSection, MovieSection, MusicSection] = plex.library.section('TV Shows')
episode_list = []
for networkName in NETWORKS:
    print(networkName)
    for show in tv_shows_section.searchShows(network=networkName):
        print(show)
```

**Comment by blacktwin** [comment]
First, to delete a playlist use `playlist.delete()`.

Second, it's actually studio not network. But you were correct to assume that network would be what you should use as thats what's in the `ALLOWED_FILTERS` for the [ShowSection](https://github.com/pkkid/python-plexapi/blob/master/plexapi/library.py#L712-L717). `network` just needs to be changed to `studio` and your search then needs to change to

```python
for networkName in NETWORKS:
    print(networkName)
    for show in tv_shows_section.searchShows(network=networkName):
        print(show)
```

If you want to create the PR for it go for it. If not I can. Up to you, it's literally one string.

**Comment by Hellowlol** [comment]
@blacktwin Still working on this?

**Comment by blacktwin** [comment]
Not really, but I can. I borked the original PR. I can't remember why/how.

**Comment by blacktwin** [comment]
Should be resolved with #518
